test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -469,6 +470,7 @@ public void startup(ICluster cluster)
                 StorageService.instance.populateTokenMetadata();
 
                 Verb.REQUEST_RSP.unsafeSetSerializer(() -> ReadResponse.serializer);
+                Verb.HINT_REQ.unsafeSetSerializer(HintMessage.DTestSerializer::new);
```

Comment:
On merge, please can you make sure the REQUEST_RSP unsafe serializer is still removed after CASSANDRA-16385 as it should no longer be needed.